src/hooks/a11y/useTrapFocus.js

```diff
@@ -0,0 +1,80 @@
+import { useState, useEffect } from 'react'
+
+const FOCUSABLE_ELEMENTS_STRING =
+  'a[href], area[href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), button:not([disabled]), iframe, object, embed, [tabindex="0"], [contenteditable]'
```

Comment:
Nit: Where's you come up with this from? It's a bit scary in web world e.g. what if something were added or removed. Maybe at least just a link back to source would be worthwhile.

Reply:
This matrix is kinda interesting for this:

https://allyjs.io/data-tables/focusable.html

Reply:
added a comment